app/styles/quiz.scss

```diff
@@ -30,5 +30,34 @@
         font-style: italic;
       }
     }
+    
+  }
+
+  &__answer {
+    .highlight__answer {
+        background-color: $highlight-color;
+    }
+  }
+}
+
+.prompt {
+  position: relative;
+  //top: 50px;
+  left: 30%;
+  margin-top: 25px;
+  text-align: center;
+  -webkit-animation: fade-in 1s;
```

Comment:
in tandem with the above mentioned change to `.prompt` you can achieve this fade by doing `transition: opacity 1s ease;` and having a class `.active { opacity: 1; }`